stages/org.osbuild.ostree.deploy.container

```diff
@@ -80,6 +83,14 @@ SCHEMA_2 = """
 }
 """
 
+def skopeo_cli(*args):
+    """Thin wrapper for running the skopeo CLI"""
+    args = list(args)
+    print("skopeo " + " ".join(args), file=sys.stderr)
+    return subprocess.run(["skopeo"] + args,
```

Comment:
There is also the `skopeo experimental-image-proxy` which is how the ostree stack fetches container metadata, and I had always envisioned generalizing it into a generic "language bindings for the containers/ Go stack" btw.

It's not hard to speak the protocol; there's a Rust version in https://github.com/containers/containers-image-proxy-rs